### DISCUSSION TITLE
Welcome to ginan Discussions!

### DISCUSSION THREAD

**Discussion by seballgeyer** [GeoscienceAustralia/ginan#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nayankumar456** [comment]
How can we insall Gian ?
Just got confused from the notes available on https://geoscienceaustralia.github.io/ginan/page.html?c=on&p=install.index

**Comment by XristiG** [comment]
Hi
I am doing an iPhD at Curtin Uni and using Ginan to obtain daily timeseries for CORS stations and Small form Geodetic Grade Receivers. I will be using PPP and base-line solutions
Ginan here is set up on a linux server so there are additional steps you have to take to get Ginan singing to your tune.

I was having download issues with incomplete files whilst using a desktop on the Uni ethernet, however downloads are working fine on the server - apart from sorting out a minor error now listed in 'general'
